sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3449,6 +3449,424 @@
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "textarea",
+          "wysiwyg": false,
+          "input": true,
+          "refreshOn": "",
+          "dataGridLabel": false,
+          "allowMultipleMasks": false,
+          "addons": [],
+          "mask": false,
+          "inputType": "text",
+          "inputMask": "",
+          "fixedSize": true,
+          "id": "e6gwlmg"
+        },
+        {
+          "label": "I Agree to the terms and conditions of the StudentAid BC Declaration",
+          "labelWidth": "",
+          "labelMargin": "",
+          "description": "",
+          "tooltip": "",
+          "shortcut": "",
+          "inputType": "checkbox",
+          "customClass": "",
+          "tabindex": "",
+          "hidden": false,
+          "hideLabel": false,
+          "autofocus": false,
+          "disabled": false,
+          "tableView": false,
+          "modalEdit": false,
+          "defaultValue": false,
+          "persistent": true,
+          "protected": false,
+          "dbIndex": false,
+          "encrypted": false,
+          "redrawOn": "",
+          "clearOnHide": true,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "allowCalculateOverride": false,
+          "validate": {
+            "required": true,
+            "customMessage": "",
+            "custom": "",
+            "customPrivate": false,
+            "json": "",
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "errorLabel": "",
+          "errors": "",
+          "key": "sinConsent",
+          "tags": [],
+          "properties": {},
+          "conditional": {
+            "show": false,
+            "when": "mode",
+            "eq": "aest-account-approval",
+            "json": ""
+          },
+          "customConditional": "",
+          "logic": [],
+          "attributes": {},
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "checkbox",
+          "name": "",
+          "value": "",
+          "input": true,
+          "placeholder": "",
+          "prefix": "",
+          "suffix": "",
+          "multiple": false,
+          "unique": false,
+          "refreshOn": "",
+          "dataGridLabel": true,
+          "labelPosition": "right",
+          "widget": null,
+          "validateOn": "change",
+          "showCharCount": false,
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "addons": [],
+          "id": "exdbzkf"
+        },
+        {
+          "label": "I Agree to the terms and conditions of the StudentAid BC Declaration",
```

Comment:
why duplicated checkbox. one in line 3672

Reply:
Since students and ministry users use the form, one checkbox is to display in read-only (This is the one shown to ministry users to prevent them from changing the value). The second checkbox is the one used by students to accept SIN consent before submitting.